gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/mapper/AlertMapper.java

```diff
@@ -191,7 +191,7 @@ private String convertStatusCode(Double statusCodeLow, Double statusCodeHigh) {
             try {
                 WebhookNotifierConfiguration webhookConfig = new WebhookNotifierConfiguration(
                     alertInput.getWebhook().getHttpMethod().getValue(),
-                    alertInput.getWebhook().getUrl()
+                    alertInput.getWebhook().getUrl().trim()
```

Comment:
i would have prefer strip but it's good for me

Reply:
done